src/Chapter2.hs

```diff
@@ -336,7 +349,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b ls
+  | a < 0 || b < 0 = []
+  | a <= b = take (b - a + 1) (drop a ls)
+  | otherwise = []
```

Comment:
Nice catch on the `a <= b`!
If you fancy fewer amount of cases, you can collapse the last one with the first:

```suggestion
  | a < 0 || b < 0 || a > b = []
  | otherwise = take (b - a + 1) (drop a ls)
```

But it is not necessary, it looks great now! 👍🏼